manifests/params.pp

```diff
@@ -97,6 +97,10 @@
 #   By default this module manages the resource Package[net-tools], if this resource is
 #   already declared on your code base, you can disable this flag.
 #
+# @param allow_insecure
+#   Disable certificate verification when connecting to SSL hosts to download packages.
+#   Defaults to false
```

Comment:
Puppet Strings docs automatically generate the defaults, so no need to write it.

```suggestion
```

Reply:
Thanks; submitted that change.